noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/note.nr

```diff
@@ -246,6 +246,40 @@ where
     plaintext_bytes
 }
 
+/// Computes an encrypted log for a note using AES-128 encryption in CBC mode.
```

Comment:
Sneaked this unrelated change in. I needed it to orient myself in the structure of the log and AI is perfect for generating these code docs.